src/json_path_test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
+import { jsonpath } from './json_path';
+
+describe('jsonpath', () => {
```

Comment:
Can you add a test like this to the gcp_auth test also?

Thanks

Reply:
done 😄